src/utils/index.js

```diff
@@ -369,6 +369,12 @@ const buildInitialPrices = (entitlements, courseRuns) => {
 const hasMastersTrack = (runTypeUuid, runTypeModes) => (!!runTypeUuid
   && !!runTypeModes[runTypeUuid] && runTypeModes[runTypeUuid].includes(MASTERS_TRACK.key));
 
+const restrictionTypeOptions = [
+  { value: 'custom-b2b-enterprise', label: 'Custom Enterprise' },
+  { value: '', label: '--------' },
```

Comment:
nit: the empty value should be first, no?

Reply:
I'll update it before merging